src/server_manager/web_app/digitalocean_account.ts

```diff
@@ -101,7 +101,7 @@ export class DigitalOceanAccount implements digitalocean.Account {
     const dropletSpec = {
       installCommand,
       size: MACHINE_SIZE,
-      image: 'docker-18-04',
+      image: 'docker-20-04',
```

Comment:
Where is this string sourced from? might be nice to comment a link here for future upgrades

Reply:
Good point. Added: https://slugs.do-api.dev/